otter/auth.py

```diff
@@ -284,6 +284,42 @@ def __hash__(self):
                      self._url, self._admin_url))
 
 
+@implementer(IAuthenticator)
+class SingleTenantAuthenticator(object):
+    """
+    An authentication handler that authenticates using a single tenant id.
+    """
+    def __init__(self, _identity_user, _identity_password, url):
+        self._identity_user = _identity_user
+        self._identity_password = _identity_password
+        self._url = url
+
+    @wait(ignore_kwargs=['log'])
+    def _auth_me(self, tenant_id, log=None):
+        if log:
+            log.msg('Authenticating as new tenant')
+        d = authenticate_single_tenant(self._url,
```

Comment:
There is an existing function `authenticate_user` in this file that does the same job as `authenticate_single_tenant`. Any reasons for not using that?